app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+import { Formik, Form as NeetoUIForm } from "formik";
```

Comment:
This is not a neetoUI module.
```suggestion
import { Formik, Form as FormikForm } from "formik";
```